src/crystal/system/win32/file_descriptor.cr

```diff
@@ -276,8 +268,14 @@ module Crystal::System::FileDescriptor
   end
 
   def self.from_stdio(fd)
```

Comment:
**question:** Is this method even still necessary? AFAIK it was only used for opening the stdio handles.

Reply:
I believe all the console-specific stuffs in the method body are still needed here

Reply:
Ah right. You're only chaning the `ORIGINAL_*` stdio handles. The main handles in `kernel.cr` still use this method.